### PR TITLE
Nov updates

### PR DESCRIPTION
**Pull request by mwasham**

### CODE REVIEW

**Comment by DawnmarieDesJardins** [comment]
Via email from Rochak

I don’t see any content around Azure Site Recovery (ASR) being added in the workshop? Are we not covering DR as part of resiliency? 

Regards
Rochak

**Comment by raj-ganapathy-msft** [comment]
Please include Availability Zones into the scenario construct.

Regards,
Raj.

**Comment by DawnmarieDesJardins** [comment]
@mwasham @hopero929 
Hi Michael,
Please move forward and finalize this update with the feedback we have received EOD today.  Assign the PR to me when it's ready for QC and merge.
Thanks, Dawnmarie

File: Whiteboard design session/WDS trainer guide - Building a resilient IaaS architecture.md [hunk 1]
```diff
@@ -440,93 +441,76 @@ Directions: Tables reconvene with the larger group to hear the facilitator/SME s
...
 -   West US 2: 172.16.0.0/16 (Domain Controller: 172.16.2.4)
 
 Network Security Groups for the West Central US Virtual Network|
 
 **Apps Tier NSGs** 
-
-|    |            |       
-|----------|:-------------:|                                                                                                           
-| Name      |       **Priority**  |  **Source**   |    **Protocol**  |  **Source Port Range** |  **Destination** |  **Dest. Port Range** |  **Action** |
+||||||||
```
Comment (DawnmarieDesJardins) [inline Whiteboard design session/WDS trainer guide - Building a resilient IaaS architecture.md]: Please check coding here.  I think these are supposed to be tables, but they are not resolving that way in the document.

Comment (ciphertxt) [reply Whiteboard design session/WDS trainer guide - Building a resilient IaaS architecture.md]: Pushed an updated version with the formatting to account for Github-flavored Markdown. (https://github.com/Microsoft/MCW-Building-a-resilient-IaaS-architecture/commit/9b9e435ac02d0a11c86baabe8281da994ba19af9)

**Pull request merged**